cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'layout-full.html' %}
+
+{# HEAD items #}
+
+{% block title -%}
+    Prepaid Agreements Search | Consumer Financial Protection Bureau
+{%- endblock title %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/prepaid-agreements/css/main.css') }}">
+{%- endblock css %}
+
+{# BODY items #}
+
+{% block content_main_modifiers -%}
+    content__1-3 prepaid-agreements prepaid-agreements-search
+{%- endblock %}
+
+{% block content_main scoped %}
+    {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
```

Comment:
Is this index page eventually intended to live under /data-research/prepaid-agreements/, instead of at /prepaid-agreements/?

Reply:
@chosak Yes, we moved the prepaid urls under /data-research in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/64f2aa21bd21a6135639605695f90bd0385a7ce7.